### PR TITLE
Aadded tabs component examples

### LINKED ISSUE #191: Add `Tabs` examples
### Subject

`Tabs`

### Description

We need to make `Tabs` examples. You may refer to https://tailwindui.com/components/application-ui/navigation/tabs when making these examples.

### Are you willing to participate in the development of this feature and create a pull request?

None

### PR DESCRIPTION
**Pull request by cidkumagai**
Closes #191  

## Description

Aadded tabs component examples.

### CODE REVIEW

File: contents/disclosure/tabs/with-icons/index.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      </Tab>
+      <Tab gap="sm">
+        <Users />
+        Team Members
+      </Tab>
+      <Tab gap="sm">
+        <CreditCard />
+        Billing
+      </Tab>
+    </Tabs>
```
Comment (taroj1205) [inline contents/disclosure/tabs/with-icons/index.tsx:24]: Please wrap the icons with `<Icon />`.

**Review by taroj1205 — COMMENTED** [review]
Thank you for your PR. Please add Japanese translation in title for `metadata.json` files you have created.

**Comment by cidkumagai** [comment]
@taroj1205 
I did fixed now.
Please re-reivew

**Review by taroj1205 — COMMENTED** [review]
Please add Japanese translations to all `metadata.json`, not just `disclosure/metadata.json`.
- `contents/disclosure/tabs/metadata.json`
- `contents/disclosure/tabs/with-badges/metadata.json`
- `contents/disclosure/tabs/with-full-width/metadata.json`
- `contents/disclosure/tabs/with-icons/metadata.json`

**Review by taroj1205 — APPROVED** [review]
LGTM

**Review by nakayan5 — APPROVED** [review]

**Pull request merged**